### PR TITLE
Fix video embeds to check proxy_url for null

### PR DESCRIPTION
**Pull request by JohnyTheCarrot**

### CODE REVIEW

**Review by advaith1 — APPROVED** [review]

File: src/Content/Embed/EmbedVideo.tsx [hunk 1]
```diff
@@ -57,11 +57,11 @@ interface EmbedVideoProps
   extends Required<Pick<APIEmbedVideo, "width" | "height">> {
   thumbnail?: APIEmbedThumbnail["url"];
   url: APIEmbedVideo["url"] | undefined;
-  proxyUrl: APIEmbedVideo["proxy_url"] | undefined;
+  proxyUrl: APIEmbedVideo["proxy_url"] | undefined | null;
```
Comment (mason-rogers) [inline src/Content/Embed/EmbedVideo.tsx:60]:
```suggestion
  proxyUrl?: APIEmbedVideo["proxy_url"];
```

Comment (JohnyTheCarrot) [reply src/Content/Embed/EmbedVideo.tsx:60]: This is [not equivalent](https://www.typescriptlang.org/docs/handbook/advanced-types.html#optional-parameters-and-properties).

webstorm64_SEtUWsFiS1

Comment (Yomanz) [reply src/Content/Embed/EmbedVideo.tsx:60]: @JohnyTheCarrot set it to undefined and not null?

Comment (JohnyTheCarrot) [reply src/Content/Embed/EmbedVideo.tsx:60]: @Yomanz what do you mean?

**Review by Yomanz — APPROVED** [review]
Splendid

**Review by mason-rogers — APPROVED** [review]

**Pull request merged**